File: app/src/main/java/com/huanli233/biliterminal2/model/OpusCard.java
package com.huanli233.biliterminal2.model;

public class OpusCard {
    public static final int TYPE_DYNAMIC = 1;
    public static final int TYPE_ARTICLE = 2;

    public String content;
    public String cover;
    public long opusId;
    public String timeText;
    public String title;

    public int type;
    public long parsedId;

    public OpusCard(int type, long id) {
        this.type = type;
        this.parsedId = id;
    }

    public OpusCard() {

    }
}

File: BiliWebApi/src/main/java/com/huanli233/biliwebapi/bean/opus/OpusCard.kt
package com.huanli233.biliwebapi.bean.opus

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import com.huanli233.biliwebapi.util.LowerCaseUnderScore
import kotlinx.parcelize.Parcelize

@Parcelize
data class OpusCard(
    @LowerCaseUnderScore val opusId: String,
    val title: String,
    val content: String,
    val cover: String,
    @LowerCaseUnderScore val timeText: String,
    @LowerCaseUnderScore val isExpired: Boolean
) : Parcelable

@Parcelize
data class OpusCardList(
    val items: List<OpusCard>,
    @LowerCaseUnderScore val hasMore: Boolean,
    val total: Int
) : Parcelable